crates/sui-network/src/randomness/mod.rs

```diff
@@ -361,6 +362,23 @@ impl RandomnessEventLoop {
             debug!("skipping received partial sigs, we already have completed this sig");
             return;
         }
+
+        // If sigs are for a future epoch, we can't fully verify them without DKG output.
+        // Save them for later use.
```

Comment:
why does self.peer_share_ids.is_none() mean it's for a future epoch?

Reply:
the only time self.peer_share_ids can be None is if update_epoch has never been called yet since the time the process started

(this means DKG is still in process for the current epoch, so it's a "future" epoch from the perspective of the RandomnessNetworkLoop)